app/models/journal.rb

```diff
@@ -1,6 +1,7 @@
 class Journal < ApplicationRecord
-
+  
   validates :title, presence: true
   validates :description, presence: true
-  
+
+  has_many :journal_entries
```

Comment:
Please study about `dependent_destroy`, `inverse_of`, etc. and let's see if we can use them in our code. Resources:
- [https://guides.rubyonrails.org/association_basics.html](https://guides.rubyonrails.org/association_basics.html)